src/bioregistry/data/bioregistry.json

```diff
@@ -70498,6 +70498,35 @@
       }
     ]
   },
+  "mirgenedb": {
+    "contact": {
+      "email": "BastianFromm@gmail.com",
+      "name": "Bastian Fromm",
+      "orcid": "0000-0003-0352-3037"
+    },
+    "contributor": {
+      "email": "palayoor.n@northeastern.edu",
+      "github": "nalikapalayoor",
+      "name": "Nalika Palayoor",
+      "orcid": "0009-0008-8406-631X"
+    },
+    "description": "mircoRNA Gene Database is a collection of animal microRNA genes. This database allows researchers to analyze mircoRNA properties like secondary structure and features in a phylogenetic context.",
+    "example": "hsa/Let-7-P1b",
+    "github_request_issue": 1393,
+    "homepage": "https://mirgenedb.org/",
+    "name": "microRNA Gene Database",
+    "pattern": "^[a-zA-Z0-9-]+/[a-zA-Z0-9-]+$",
```

Comment:
Somewhat inconveniently from the perspective of the `uri_format`, it looks like the actual IDs use `-` rather than `/`. We should prioritize curating the ID accurately so I would suggest using `-` here. Then, the `uri_format` could be updated to something like https://mirgenedb.org/browse?qtype=mgid&org=ALL&query=$1 which is a search endpoint but correctly resolves these IDs.

![image](https://github.com/user-attachments/assets/ac445f3c-60e7-4bf8-82b6-208220b79e6d)